homeassistant/components/adax/climate.py

```diff
@@ -161,6 +173,14 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
     async def async_update(self) -> None:
         """Get the latest data."""
         data = await self._adax_data_handler.get_status()
-        self._attr_target_temperature = data["target_temperature"]
         self._attr_current_temperature = data["current_temperature"]
         self._attr_available = self._attr_current_temperature is not None
+        if data["target_temperature"] == 0:
```

Comment:
```suggestion
        if (target_temp := data["target_temperature"]) == 0:
```

Reply:
Not familiar with this coding style. Presumably it sets target_temp inside the if-statement and uses that down the line? If it is only cosmetic I am fine with the change.

Reply:
It assigns a variable as it computes the statement and hence we can use it further down the code.
See for some explanations https://realpython.com/python-walrus-operator/